acquia_lift_profiles/js/acquia_lift_profiles.js

```diff
@@ -51,6 +51,7 @@ var _tcwq = _tcwq || [];
             }
           }
         }
+        $(document).trigger('retrievedSegments', [segments]);
```

Comment:
Can we rename this event to acquiaLiftStoredSegments? That namespaces it and also matches what is actually happening here, which is that we are storing segments in the cache.